Makefile

```diff
@@ -1,3 +1,5 @@
+GENNI_VERSION=0.1.2
+
 # Format all files using autoflake / isort / black
 remove_unused_imports:
 	autoflake genni --recursive --in-place --remove-unused-variables --remove-all-unused-imports --ignore-init-module-imports
```

Comment:
not on requirements list/inherent in the mac env...

Reply:
Not sure what you mean by this. This is to reinstall it easily while testing. To be honest could remove this.

Reply:
For the warnings, I can fix them by using the correct yaml load (`safe_load` vs `load`). Will update at some point.